api/localv1/api.proto

```diff
@@ -14,20 +14,33 @@
 
 syntax = "proto3";
 
-package localnetv1;
+package localv1;
 
-option go_package = "sigs.k8s.io/kpng/server/pkg/api/localnetv1";
+option go_package = "sigs.k8s.io/kpng/api/localv1";
 
-service Endpoints { // FIXME rename to Local
+service Sets {
     // Returns all the endpoints for this node.
     rpc Watch (stream WatchReq) returns (stream OpItem);
 }
 
+// any time were watching for the local model, we need to send the
+// "location" from where we are watching.
 message WatchReq {
     // NodeName of the requester
     string NodeName = 1;
 }
+enum Set {
+    UnknownSet = 0;
+
+
+    ServicesSet = 1;
+    EndpointsSet = 2;
 
+    // FIXME move to a 3rd generic proto ???
```

Comment:
Hrm can't this just live in `globalv1`?

Reply:
It can it it does but... these global fields aren't referenced in the local data model.  hence the. fixme. I f nothing else, it helps w/ long term understanding of what we were thinking when we included these unused fields in the local proto